Dockerfile.stream8

```diff
@@ -16,4 +18,17 @@ RUN dnf install -y \
         make \
         rpm-build \
         tar
-COPY containerroot /
+
+COPY containerroot /workspace/
```

Comment:
Why would you copy the container root to the workspace folder? What's the reason behind this?

Reply:
To be honest I am not really sure why this empty folder is there.

Reply:
Normally, this is where you would place files that you want in the container. The containerroot folder being `/` and everything in there mirroring the structure you would want inside the container.

Reply:
I mean I understand the concept, but isn't it better to copy them specifically. Anyway this patch is probably not the best place to discuss this. So I'll keep it as it was.